moo.js

```diff
@@ -544,7 +544,24 @@
     }
   }
 
+  Lexer.prototype.makeEOF = function(type) {
```

Comment:
Is `makeEOF` useful for something other than `formatError` (to which the user can just pass `null` or `undefined`)? It seems more like an implementation detail.

Furthermore, aren't `line` and `col` inconsistent with `offset` when the lexer isn't actually at EOF? Should you be allowed to call `makeEOF` in that scenario?

Reply:
> aren't `line` and `col` inconsistent with `offset` when the lexer isn't actually at EOF?

That's a great point; I'm not sure what I was thinking.

Reply:
There are some styles of parser where it's useful to have an EOF token; I think that was the idea here. However I haven't yet needed `makeEOF()` in practice, so I'm happy to remove it.